idmapping-rest/pom.xml

```diff
@@ -175,6 +175,36 @@
 			<version>${embedded-redis.version}</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.testcontainers</groupId>
+			<artifactId>testcontainers</artifactId>
+			<version>1.17.6</version>
+			<scope>test</scope>
```

Comment:
could you please introduce a property (e.g. testcontainers.version) and use this in version like ${testcontainers.version}. The rationale behind this is that we will have just one place to update when we have to update the version.